src/librustc_typeck/lib.rs

```diff
@@ -261,9 +261,10 @@ fn check_start_fn_ty(ccx: &CrateCtxt,
                     match it.node {
                         hir::ItemFn(_,_,_,_,ref ps,_)
                         if ps.is_parameterized() => {
-                            struct_span_err!(tcx.sess, start_span, E0132,
+                            let sp = ps.span().unwrap();
```

Comment:
This looks okay, though I'm not 100% sure there's always a span if it's .is_parameterized(). 

We could support a fallback with something like:

```rust
let sp = if let Some(span) = ps.span() { span } else { start_span };
```